### PR TITLE
implement list of groceries with checkboxes

### LINKED ISSUE #33: List component
## User story
As a **user**,
I want the users **to be able to check their groceries and mark it available or unavailable by using checkboxes.**  

## Notes/Links
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch
https://stackoverflow.com/questions/40302344/different-handlechange-functions-for-every-single-element-in-component-react

## Acceptance criteria
- [x] Use fetch and get requests for rendering groceries from database
- [x] rendering groceries by type
- [x] Create custom hook for adding checkboxes
- [x] Implementing icons
- [x] Changing icons by using handleChange() 
- [x] Page integration

### PR DESCRIPTION
**Pull request by ReneSchmitz**
### Link/s
https://fabstr-pr-34.herokuapp.com/

### Closed Issue/s
close #33

### CODE REVIEW

File: client/src/components/Checkbox.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+import Checkbox from "../assets/icon/checkbox.svg";
+import CheckboxChecked from "../assets/icon/checkboxChecked.svg";
+
+export default function GetCheckBox() {
+  const [ischecked, setIsChecked] = useState(false);
```
Comment (Sina-Sche) [inline client/src/components/Checkbox.js:6]:
```suggestion
  const [isChecked, setIsChecked] = useState(false);
```

for better readability, I would use camelCase for "isChecked"

File: client/src/components/Checkbox.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  const CheckboxSrc = !ischecked ? Checkbox : CheckboxChecked;
+  const CheckboxAlt = !ischecked ? "Checkbox" : "Checkbox is ticked";
```
Comment (Sina-Sche) [inline client/src/components/Checkbox.js:8]:
```suggestion
  const CheckboxSrc = !isChecked ? Checkbox : CheckboxChecked;
  const CheckboxAlt = !isChecked ? "Checkbox" : "Checkbox is ticked";
```

same as above ;)

**Review by Sina-Sche — COMMENTED** [review]
Just a very small suggestion, other than that looks good to me :) 

About your problem with the deployment: I had the same issue and solved it by changing the order of `app.use `statements in the server.js (so this is concerning your previos PullRequest - I just commented on it now, you should be able to see it here: 

https://github.com/ReneSchmitz/fabstr/pull/30/files#diff-a4c65ede64197e1a112899a68bf994485b889c4b143198bac4af53425b38406fR10-R12

File: client/src/components/Checkbox.js (hunk 1 above)
Comment (ReneSchmitz) [reply client/src/components/Checkbox.js:6]: Your absolutely right! Thx for noticing!

File: client/src/components/Checkbox.js (hunk 2 above)
Comment (ReneSchmitz) [reply client/src/components/Checkbox.js:8]: Thx!! 🥳

**Pull request merged**